src/components/TextSize.js

```diff
@@ -1,32 +1,56 @@
 import { h } from 'preact'
-import { useI18n, useSoftkey, useArticleTextSize } from 'hooks'
+import { useEffect, useContext } from 'preact/hooks'
+import { useI18n, useSoftkey } from 'hooks'
 import { articleTextSize } from 'utils'
+import { FontContext } from 'contexts'
+
+let originalTextSize
 
 export const TextSize = ({ close }) => {
   const i18n = useI18n()
-
-  const [textsize, setTextSize] = useArticleTextSize()
-  const { onKeyArrowLeft, onKeyArrowRight } = articleTextSize.getSoftkeyEffect(setTextSize)
+  const { textSize, setTextSize } = useContext(FontContext)
   const sliderValue = ['0', '16.6', '33.2', '49', '66.6', '83.2', '94']
 
+  const onKeyFixedArrowLeft = () => {
+    articleTextSize.adjust(-1)
+    setTextSize(articleTextSize.get())
+  }
+
+  const onKeyFixedArrowRight = () => {
+    articleTextSize.adjust(1)
+    setTextSize(articleTextSize.get())
+  }
+
+  const onKeyBackspace = () => {
+    const currentTextSize = articleTextSize.get()
+
+    articleTextSize.adjust(originalTextSize - currentTextSize)
+    setTextSize(articleTextSize.get())
+    close()
+  }
+
+  useEffect(() => {
+    originalTextSize = articleTextSize.get()
+  }, [])
+
   useSoftkey('TextSize', {
     center: i18n('softkey-ok'),
     onKeyCenter: close,
-    onKeyBackspace: close,
-    onKeyArrowLeft,
-    onKeyArrowRight
+    onKeyBackspace,
+    onKeyFixedArrowLeft,
+    onKeyFixedArrowRight
```

Comment:
These we should use `onKeyArrowLeft` and `onKeyArrowRight`

FixedArrow is not this purpose

Reply:
What is the difference between `onKeyArrowRight/Left` and `onKeyFixedArrowRight/Left`?

Reply:
`onKeyArrowRight/Left` will be switched if `direction=rtl`

`onKeyFixedArrowRight/Left` doesn't no matter the `direction` value, one example is `Table.js`

Reply:
Gotcha that makes sense, thanks. Will update handlers to `onKeyArrowRight/Left` soon